test/cql-pytest/cassandra_tests/validation/operations/compact_storage_test.py

```diff
@@ -3271,7 +3271,7 @@ def testFilteringOnCompactTablesWithoutIndicesAndWithMaps(cql, test_keyspace):
                        row(1, 3, {6: 2}),
                        row(2, 3, {7: 1}))
 
-            assertInvalidMessage(cql, table, 'filtering',
+            assertInvalidMessage(cql, table, 'FILTERING',
```

Comment:
Please confirm that this doesn't break this test on Cassandra (try `test/cql-pytest/run-cassandra cassandra_tests/validation/operations/compact_storage_test.py::testFilteringOnCompactTablesWithoutIndicesAndWithMaps).
The test suggests that Cassandra used the word "filtering" here, not "FILTERING".

If you need to support both, you can try for example
`assert_invalid_message_re(cql, table, 'filtering|FILTERING')`

Reply:
sure. i will run cassandra against the changed tests.

Reply:
```console
$ git rev-list -n 1 --pretty=short HEAD
commit cbc269e9cf5a387f0bf1d14bc2cf7fb5e8686333
Author: Kefu Chai <kefu.chai@scylladb.com>

    cql3/restrictions/statement_restrictions: s/allow filtering/ALLOW FILTERING/
$ cd test/cql-pytest
$ ./run-cassandra  cassandra_tests/validation/operations/compact_storage_test.py::testFilteringOnCompactTablesWithoutIndicesAndWithMaps
...
Cassandra tests pass
$ ./run-cassandra cassandra_tests/validation/operations/select_single_column_relation_test.py::testInvalidSliceRestrictionOnPartitionKey
...
Cassandra tests pass
$ ./run-cassandra cassandra_tests/validation/operations/select_test.py::testAllowFilteringOnPartitionKeyWithDistinct
...
Cassandra tests pass
$ ./run-cassandra cassandra_tests/validation/operations/select_test.py::testAllowFilteringOnPartitionAndClusteringKey
...
Cassandra tests pass
$ ./run-cassandra cassandra_tests/validation/operations/select_test.py::testContainsOnPartitionKey
...
Cassandra tests pass
```

the error message returned by Cassandra is

> Cannot execute this query as it might involve data filtering and thus may have unpredictable performance. If you want to execute this query despite the performance unpredictability, use ALLOW FILTERING

so, both "filtering" and "ALLOW FILTERING" match with this error message. we could make it less tolerant by using "ALLOW FILTERING" though.

Reply:
Oh, I didn't realize the error message had the word "filtering" twice with two different capitalizations :-) Ok.